### DISCUSSION TITLE
updated from 1.32.0 to 1.32.4 and "Log in with device is no longer functioning"

### DISCUSSION THREAD

**Discussion by estranged-networks** [dani-garcia/vaultwarden#5192] [Q&A] [answered]
Hi, I did the most recent upgrade for VW and all went well. But when i try log in with device option I get the correct Fingerprint phrase sent to my device but if i Tap on it , i get a notification box that reads  "AuthRequest doesn't exist"  and on the server side logs read...
[vaultwarden::api::core::accounts][ERROR] AuthRequest doesn't exist. User uuid's do not match
[response][INFO] (get_auth_request) GET /api/auth-requests/<uuid> => 400 Bad Request

I guess my question is , does anyone know what the security updates in this minor update were and if its safe to revert back to v 1.32.0 to restore the "log in with device " functionality ?  ( I am aware another user has posted a similar issue yesterday but that is I beleive to do with push notifications not working)

**Comment by BlackDex** [comment, marked as answer]
If anything I would suggest to use `testing` right now to solve it until we release a new version shortly.

**Reply by estranged-networks** [reply]
same issue with the testing image im affraid

**Reply by estranged-networks** [reply]
scratch that, It is working with Version 1.32.4-adb21d5c

**Comment by Gerardv514** [comment]
> "AuthRequest doesn't exist" and on the server side logs read...
> [vaultwarden::api::core::accounts][ERROR] AuthRequest doesn't exist. User uuid's do not match
> [response][INFO] (get_auth_request) GET /api/auth-requests/ => 400 Bad Request

I am seeing the same errors in my logs. I am running latest 1.32.7, and just came to notice that when I swipe down on the vault on ios app (2024.12.0), sync isn't occurring. When I go to ios app settings, other I can see the last sync time didn't change. If I click sync now in the ios settings/other, it syncs. I added an test vualt item in chrome extension, saw it in the web vault and then confirmed it did not come across to the ios device (even with swiping down) until I went into settings and performed a sync there. Also recently I've been updating some entries to include passkeys and would notice after adding the passkey I had to close the ios app out and re-open it to see the entry updated.

I also just regenerated the Push ID & Key, which didn't resolve. Looking in my logs I had these errors back in July then it stopped and then started around November 30th. 

I also took the ios BW app and logged into Bitwarden Cloud, when I pull down on the vault to refresh the vault doesn't refresh.  When I go into settings and click 'sync now' the time of sync updates and it syncs; so I'm seeing this on the cloud & vw side.

Strange, but not surprised if the app is where the issue is, wondering if anyone else can confirm they're seeing the same with swiping down and syncing. 

Posting this here since the error messages mentioned matched, and Log in with device is also dependent on Sync function (to the best of my knowledge).

**Reply by Gerardv514** [reply]
Pull down Sync is broke in 2024.12.

The AuthRequest errors are not part of sync issue, although they appeared around same time sync stopped working.

https://github.com/bitwarden/ios/issues/742
